scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -53,8 +53,6 @@ class RemoveUnused(config: RemoveUnusedConfig)
         diagnostic.message.startsWith("local") &&
         diagnostic.message.endsWith("is never used")) {
         isUnusedTerm += diagnostic.position
-      } else {
-        () // ignore
```

Comment:
this didn't seem to do much, so I've just removed it

Reply:
It's was more for stylistic reasons but I don't feel strongly about it. I don't like having such a large `if else` without a closing `else`